### PR TITLE
List View Functionality for Music

### PR DESCRIPTION
**Pull request by jsinger32**
#### Description
There is now a functional UI for music listening. It is a scrollable list view that is made up of buttons that can send users anywhere with the existing navigation system. Work will need to be done as far as making this UI presentable, but it is now functional.

#### Screenshots (if appropriate):
image

#### Notes
The left half of the screenshot is what is visible upon entering the module
The right half of the screenshot is what the module looks like upon scrolling to the bottom
Tapping on the text on the screen sends users to other modules (For the moment, see checklist)

#### Types of Changes

- [ ] New feature: Musical Works module is now more than blank text. A functional 'SliverList' widget has been applied that is made up of TextButton widgets that will allow for Navigation to the In-App Music Player a la [OM2329-34](https://jib-2329.atlassian.net/browse/OM2329-34)

#### Checklist

- [ ] This UI needs serious visual work, as it is effectively stock Material at the moment
- [ ] Navigation code will need to be updated once the In-App Music Player is developed
- [ ] Database of songs and their accompanying indexes in that database will need to be implemented into this new code

### CODE REVIEW

**Comment by jsinger32** [comment]
Testing with removing the 'AppBar' globally, as seen on the homepage
image

Also experimenting with adding images to the music module, with an image added to the 'FlexibleSpaceBar' as seen here:
image

**Comment by Alex-Laney** [comment]
I like the Background image for the text, but it does cause difficulty reading the text. Maybe add a background behind the text and back button so you can see them over the image?

File: lib/Musical-Works/Musical-Works.dart [hunk 1]
```diff
@@ -6,19 +6,125 @@ class WorksScreen extends StatelessWidget {
...
+          CircularMenuItem(
+            boxShadow: const [],
+            color: Colors.red[iconColor],
+            icon: Icons.book,
+            iconColor: Colors.black,
+            onTap: () => Navigator.pushNamed(context, '/terms'),
+          ),
+        ],
+      );
+    }
```
Comment (SpeakerRegional) [inline lib/Musical-Works/Musical-Works.dart:50]:
```suggestion
```
This is not used anymore and instead is currently being put in its own separate file.

File: lib/Musical-Works/Musical-Works.dart [hunk 2]
```diff
@@ -6,19 +6,125 @@ class WorksScreen extends StatelessWidget {
...
-      body: Center(
-        child: Column(
-          mainAxisSize: MainAxisSize.min,
-          children: const <Widget>[Text('ExampleWorksInformationHere')],
+        // appBar: AppBar(
+        //   title: const Text('Classical Works'),
+        // ),
+        body: CustomScrollView(
+          slivers: <Widget>[
+            SliverAppBar(
```
Comment (SpeakerRegional) [inline lib/Musical-Works/Musical-Works.dart:61]:
```suggestion
            const SliverAppBar(
```
Was suggested by an IDE to improve performance.

File: lib/main.dart [hunk 3]
```diff
@@ -40,9 +40,6 @@ class MainScreen extends StatelessWidget {
         ElevatedButton.styleFrom(textStyle: const TextStyle(fontSize: 20));
 
     return Scaffold(
-      appBar: AppBar(
-        title: const Text('Home Page'),
-      ),
```
Comment (SpeakerRegional) [inline lib/main.dart:45]: Was this change intentional?

File: lib/main.dart [hunk 4]
```diff
@@ -120,9 +117,6 @@ class ListenScreen extends StatelessWidget {
     final ButtonStyle style =
         ElevatedButton.styleFrom(textStyle: const TextStyle(fontSize: 20));
     return Scaffold(
-        appBar: AppBar(
-          title: const Text('Listen'),
-        ),
```
Comment (SpeakerRegional) [inline lib/main.dart:125]: Same with this one. The user seems to not be able to go back to the previous screen.

**Review by SpeakerRegional — CHANGES_REQUESTED** [review]
Works as intended after deleting the suggested code block. Also, left some small suggestions and questions.

File: lib/main.dart (hunk 3 above)
Comment (jsinger32) [reply lib/main.dart:45]: No, I forgot to undo this. Was testing it out.

File: lib/main.dart (hunk 4 above)
Comment (jsinger32) [reply lib/main.dart:125]: No, I forgot to undo this. Was testing it out.

**Review by SpeakerRegional — APPROVED** [review]
Looks good now.
